examples/autocomplete.amp.html

```diff
@@ -25,8 +26,148 @@
+      <amp-autocomplete filter="substring" max-entries="10" min-characters="0"  src="autocomplete-cities.example.json">
+          <input type="text" name="input5" required>
+      </amp-autocomplete>
+      <input name="submit-button" type="submit" value="Submit">
+    <div submit-success>
+        <template type="amp-mustache">
+            Success! {{input5}}
+        </template>
+    </div>
+    </form>
+
+    <p>submit-on-enter</p>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete filter="substring"
+        src="autocomplete-cities.example.json" submit-on-enter>
+          <input type="text" name="input6" required>
+      </amp-autocomplete>
+      <input name="submit-button" type="submit" value="Submit">
+    <div submit-success>
+        <template type="amp-mustache">
+            Success! {{input6}}
+        </template>
+    </div>
+    </form>
+
+    <h2>Filters</h2>
+    <p>substring</p>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete filter="substring" min-characters="0"
+        src="autocomplete-cities.example.json" submit-on-enter>
+          <input type="text" name="input7" required>
+      </amp-autocomplete>
+      <input name="submit-button" type="submit" value="Submit">
+    <div submit-success>
+        <template type="amp-mustache">
+            Success! {{input7}}
+        </template>
+    </div>
+    </form>
+
+    <p>prefix</p>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete filter="prefix" min-characters="0"
+        src="autocomplete-cities.example.json" submit-on-enter>
+          <input type="text" name="input8" required>
+      </amp-autocomplete>
+      <input name="submit-button" type="submit" value="Submit">
+    <div submit-success>
+        <template type="amp-mustache">
+            Success! {{input8}}
+        </template>
+    </div>
+    </form>
+
+    <p>token-prefix</p>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete filter="token-prefix" min-characters="0"
+        src="autocomplete-cities.example.json" submit-on-enter>
+          <input type="text" name="input9" required>
+      </amp-autocomplete>
+      <input name="submit-button" type="submit" value="Submit">
+    <div submit-success>
+        <template type="amp-mustache">
+            Success! {{input9}}
+        </template>
+    </div>
+    </form>
+
+    <p>none</p>
+    <form method="post" action-xhr="/form/echo-json/post" target="_blank">
+      <amp-autocomplete filter="none" min-characters="0"
+        src="autocomplete-cities.example.json"
+        [src]="manualFilterData" submit-on-enter>
+          <input type="text" name="input9" required 
+            on="input-debounced:AMP.setState({ manualFilterData: 
+              { 'items' : [event.value + ' 1', event.value + ' 2', event.value + ' 3'] } })">
```

Comment:
This is an interesting case for filtering with bind and `filter="none"`. I was considering another case, like the paged list amp by example https://ampbyexample.com/advanced/paged_list/ with string concatenation using a url string and `event.value` vs a raw object.